scipy/stats/_multivariate.py

```diff
@@ -4275,3 +4279,497 @@ def rvs(self, size=1, random_state=None):
     method_frozen.__doc__ = doccer.docformat(method.__doc__,
                                              mvt_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, mvt_docdict_params)
+
+
+_mhg_doc_default_callparams = """\
+m : array_like
+    The number of each type of object in the population.
+    That is, :math:`m[i]` is the number of objects of
+    type :math:`i`.
+n : array_like
+    The number of samples taken from the population.
+"""
+
+_mhg_doc_callparams_note = \
+"""`m` must be an array of positive integers. If the quantile
```

Comment:
`flake8` complains about the indentation here.  Let's follow the style used everywhere else.
```suggestion
_mhg_doc_callparams_note = """\
`m` must be an array of positive integers. If the quantile
```